_includes/Formulas/scope-loop.md

```diff
@@ -0,0 +1,4 @@
+
+#### Loop Over Variable Step Scope
+
+Loop Over Variable steps make available the current object being processed and the index to each step executed inside of that loop.  For example, if we have a `loop` step named `looper`, any steps that are run inside of that loop would have access to `looper.index` and `looper.entry`.
```

Comment:
@jjwyse any suggestions for an example? I added this but don't think it's quite enough:

Example references to Loop scope:

* `{$steps.myLoopStep.entry.id}`
* `{$steps.myLoopStep.index}`